### PR TITLE
LPD-20247 Fixed Contact us text and Solutions Search template

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 1]
```diff
@@ -44,8 +45,8 @@
 	.solution-search-results-card {
 		border-radius: 10px;
 		border: 1px solid #E7EFFF;
-		min-height: 456px;
-		min-width: 293px;
+		height: 462px;
+		width: 293px;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:49]: below overflow

**Review by kevenleone — APPROVED** [review]
Looks good @welitonpe , thank you

Could you just reorder what I mentioned and please can you cherry pick this commit?https://github.com/brianchandotcom/liferay-portal/pull/148149/commits/f9fec2542940a3c38c142498598f1b4a91eff091

**Comment by kevenleone** [comment]
Thanks @welitonpe

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward:force

**Pull request closed without merging**